src/net/rpc.rs

```diff
@@ -2384,6 +2409,91 @@ impl ConversationHttp {
         response.send(http, fd).and_then(|_| Ok(stream))
     }
 
+    /// Handle a GET health.
+    /// A node is considered healthy if it is up to the max height of its bootstrap nodes.
+    /// This endpoint also returns what percent of the max height the node height is at.
+    fn handle_get_health<W: Write>(
+        http: &mut StacksHttp,
+        fd: &mut W,
+        req: &HttpRequestType,
+        network: &PeerNetwork,
+        network_id: u32,
+        peerdb: &PeerDB,
```

Comment:
I'm curious why these two arguments (`peerdb` and `inv_state_opt`) are explicit arguments? This data already lives within the `network`.